lib/rack/tracker.rb

```diff
@@ -55,7 +55,11 @@ def html?; @headers['Content-Type'] =~ /html/; end
 
     def inject(env, response)
       @handlers.each(env) do |handler|
-        response.gsub!(%r{</#{handler.position}>}, handler.render + "</#{handler.position}>")
+        if handler.container_position == :closing
+          response.gsub!(%r{</#{handler.container_tag}>}, handler.render + "</#{handler.container_tag}>")
```

Comment:
I'm with you on the response.gsub part,... but I think the behavior what comes first/last (the position) should be inside of the handler. 

btw. ` </#{handler.container_tag}> ` doesn't match elements with attributes right?